src/components/About/About.jsx

```diff
@@ -1,29 +1,27 @@
 import React, { Component } from "react";
-import Card from "react-md/lib/Cards/Card";
-import CardText from "react-md/lib/Cards/CardText";
+import { Tile } from "carbon-components-react/lib/components/Tile";
 import UserLinks from "../UserLinks/UserLinks";
 import config from "../../../data/SiteConfig";
 import "./About.scss";
+import "carbon-components/scss/components/tile/_tile.scss";
 
 class About extends Component {
   render() {
     return (
       <div className="about-container md-grid mobile-fix">
-        <Card className="md-grid md-cell--8">
-          <div className="about-wrapper">
-            <img
-              src={config.userAvatar}
-              className="about-img"
-              alt={config.userName}
-            />
-            <CardText>
-              <p className="about-text md-body-1">
-                {config.userDescription}
-              </p>
-            </CardText>
-            <UserLinks labeled config={config} />
-          </div>
-        </Card>
+				<Tile>
```

Comment:
- [ ] Can you reformat the indentation in the `Tile` code to be 2 spaces instead of tabs please?